connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/LoggingResource.java

```diff
@@ -59,6 +60,7 @@ public class LoggingResource {
      */
     @GET
     @Path("/")
+    @Operation(summary = "List the current loggers that have their levels explicitly set and their log levels")
     public Response listLoggers() {
```

Comment:
The `Response` return types in this and other Resource classes means there are no `schemas` generated, which seems less than ideal.

Reply:
Yes this is not great. I think this should be fixable without requiring a KIP, but let's do this in a separate PR